www/pages/user/[id]/home.js

```diff
@@ -13,21 +13,27 @@ import styles from '../../../styles/pages/user/[id]/home.module.css';
 import { notify } from '../../../store/features/snackbar';
 
 const User = ( props ) => {
+
+  const filtersConstuctor = { owners: [], name: '' };
+
   const [ loading, setLoading ] = useState( true );
-  const [ ownedMeetings, setOwnedMeetings ] = useState([]);
-  const [ participantMeetings, setParticMeetings ] = useState([]);
+  const [ meetings, setMeetings ] = useState([]);
+  const [ filters, setFilters ] = useState( filtersConstuctor );
+  const [ meetingCount, setMeetingCount ] = useState( 0 );
+  const [ skip, setSkip ] = useState( 0 );
 
   const user = useSelector( ( state ) => state.user );
 
   async function load() {
     try {
       const res = await Promise.all([
-        client.get( `participant/meetings/${ user.email }` ),
-        client.get( `user/meetings/${ user._id }` )
+        client.get( `/meeting/?skip=${ skip }&limit=14`, {
+          params: { filters }
+        })
       ]);
 
-      setParticMeetings( res[ 0 ].data );
-      setOwnedMeetings( res[ 1 ].data );
+      setMeetings( res[ 0 ].data.meetings );
+      setMeetingCount( res[ 0 ].data.count );
```

Comment:
We should probably modify the api to only return the first element of the array. I feel like the array isn't what people would expect.

Reply:
Its the use of promise all doing that, Nothing like using promise all for one promise